crypto/err/openssl.txt

```diff
@@ -3088,6 +3088,7 @@ SSL_R_EXTENSION_NOT_RECEIVED:279:extension not received
 SSL_R_EXTRA_DATA_IN_MESSAGE:153:extra data in message
 SSL_R_EXT_LENGTH_MISMATCH:163:ext length mismatch
 SSL_R_FAILED_TO_INIT_ASYNC:405:failed to init async
+SSL_R_FETCH_FAILED:295:fetch failed
```

Comment:
Could the error be a little more descriptive? Something like "algorithm fetch failed" or "evp algorithm fetch failed".

Reply:
I made it "algorithm fetch failed"